src/modules/checkout/hooks/use-cart.ts

```
@@ -1,37 +1,53 @@
+  const totalItems = useMemo(() => cartItems.length, [cartItems]);
+
+  const toggleProductToCart = useCallback(
+    (productId: string) => {
+      console.log('toggleProductToCart', productId, cartItems);
+      if (cartItems.includes(productId)) {
+        removeProductFromCart(tenantSlug, productId);
+      } else {
+        addProductToCart(tenantSlug, productId);
+      }
+    },
+    [cartItems, tenantSlug, removeProductFromCart, , addProductToCart]
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)